spec/core/matchers/toHaveSpyInteractions.js

```diff
@@ -0,0 +1,57 @@
+describe('toHaveSpyInteractions', () => {
```

Comment:
This spec file doesn't run because its name doesn't end in Spec.js. If I change the name, all the specs fail.

Reply:
Renamed file to add Spec

Reply:
I've also refactored the tests to rely on jasmineUnderTest with `let matcher = jasmineUnderTest.matchers.toHaveSpyInteractions();` - currently one test is still failing, but the rest are resolved.

Reply:
The remaining test has been solved and all tests are passing now.